OrbitGl/TimeGraph.cpp

```diff
@@ -308,9 +320,8 @@ void TimeGraph::ProcessTimer(const TimerInfo& timer_info, const FunctionInfo* fu
     capture_max_timestamp_ = timer_info.end();
   }
 
-  if (function != nullptr && function->type() == FunctionInfo::kOrbitTrackValue) {
-    ProcessValueTrackingTimer(timer_info);
-    return;
+  if (function != nullptr && function->type() != FunctionInfo::kNone) {
```

Comment:
This confuses me a bit. It looks like ProcessOrbitFunctionTimer only handles manual instrumentation timers, but the naming here suggests that any timer with a proper function->type() (anything that is not kNone) is processed by that function. But regular timers for function calls are processed below.

Reply:
I guess that is how it's intended (the types are only meaningful for manual instrumentation). It would confuse me less if kNone were actually called kRegular, or "type" would be called "orbit_type".

Reply:
I changed type to orbit_type.